### PR TITLE
Use bitmap subscr

### PR DESCRIPTION
**Pull request by siehputz**
Added code in bmp/indexed.py to swap the red and green bytes. Also added a color whell image to images and an example script to load it.

### CODE REVIEW

File: adafruit_imageload/bmp/indexed.py [hunk 1]
```diff
@@ -47,7 +47,9 @@ def load(file, width, height, data_start, colors, color_depth, *, bitmap=None, p
 
         file.seek(data_start - colors * 4)
         for value in range(colors):
-            palette[value] = file.read(4)
+            c_bytes = file.read(4)
+            # Need to swap red & green bytes (bytes 0 and 2)
```
Comment (tannewt) [inline adafruit_imageload/bmp/indexed.py]: I think this is actually swapping red and blue.

**Review by tannewt — CHANGES_REQUESTED** [review]
Let me know if you need help with lint. It can be pretty picky.

File: adafruit_imageload/bmp/indexed.py (hunk 1 above)
Comment (siehputz) [reply adafruit_imageload/bmp/indexed.py]: You are correct - I've updated the comment

File: adafruit_imageload/bmp/indexed.py [hunk 2]
```diff
@@ -47,7 +47,12 @@ def load(file, width, height, data_start, colors, color_depth, *, bitmap=None, p
 
         file.seek(data_start - colors * 4)
         for value in range(colors):
-            palette[value] = file.read(4)
+            c_bytes = file.read(4)
+            # Need to swap red & blue bytes (bytes 0 and 2)
+            palette[value] = bytes(b''.join([c_bytes[2:3],
+                                             c_bytes[1:2],
+                                             c_bytes[0:1],
+                                             c_bytes[3:1]]))
```
Comment (matt-land) [inline adafruit_imageload/bmp/indexed.py:55]: faster?
```
            palette[value] = bytes([c_bytes[2], c_bytes[1], c_bytes[0], c_bytes[3]]) 
```

**Review by tannewt — APPROVED** [review]
Thank you @DavePutz and @siehputz! We can address any further comments with new PRs.

**Pull request merged**